<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labeHeaderTest">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 italic 48pt &quot;Z003&quot;;</string>
     </property>
     <property name="text">
      <string>Typing Tutorial</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizonDestailsTest">
     <item>
      <widget class="QLabel" name="labelUserNameTest">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTimeTest">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCPMTest">
       <property name="text">
        <string>CPM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCharTest">
       <property name="text">
        <string>Char</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelWPMTest">
       <property name="text">
        <string>WPM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelWordTest">
       <property name="text">
        <string>Words</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerUpperTest">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>199</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizonCharTest">
     <item>
      <widget class="QLabel" name="labelChar_1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_Main">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.525, y1:1, x2:0.509804, y2:0, stop:0 rgba(79, 231, 48, 177), stop:0.5 rgba(55, 163, 211, 224), stop:0.995098 rgba(0, 255, 67, 217));
font: 16pt &quot;Noto Sans&quot;;</string>
       </property>
       <property name="text">
        <string>typing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_11">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_12">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_13">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_14">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_15">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_16">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_17">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_18">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChar_19">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerLowerTest">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>199</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
